editor/editor_settings.cpp

```diff
@@ -1483,17 +1483,22 @@ Ref<Shortcut> EditorSettings::get_shortcut(const String &p_name) const {
 	// Use the first item in the action list for the shortcut event, since a shortcut can only have 1 linked event.
 
 	Ref<Shortcut> sc;
+	bool no_shortcut = false;
 	const Map<String, List<Ref<InputEvent>>>::Element *builtin_override = builtin_action_overrides.find(p_name);
 	if (builtin_override) {
```

Comment:
Try to avoid nested `if`s where possible, if they can be simplified without affecting the logic. In this case the `no_shortcut` local variable is not required since it it not needed further down (see my comment about that section)
```suggestion
	if (builtin_override && !builtin_override->get().is_empty()) {
```